portal-impl/test/unit/com/liferay/portal/util/PortalImplEscapeRedirectTest.java

```diff
@@ -149,12 +150,6 @@ public void testEscapeRedirectWithDomains() throws Exception {
 			_portalImpl.escapeRedirect(
 				"https://liferay.com:8080/a/b;c=d?e=f&g=h#x=y"));
 
-		// Allow request host header
-
-		Assert.assertEquals(
-			"https://request-host:8080",
-			_portalImpl.escapeRedirect("https://request-host:8080"));
-
```

Comment:
I think it is important to put this below a separate comment because it is very different in nature to configured virutal hosts. Here we are acting on the client's Host header.